data_layer/avail/avail.go

```diff
@@ -162,39 +179,44 @@ func (a *Avail) CheckDABalance() ([]keys.NotFundedAddressData, error) {
 	return nil, nil
 }
 
-func (a *Avail) getBalance() (availtypes.U128, error) {
-	if a.client == nil {
-		client, err := gsrpc.NewSubstrateAPI(a.RpcEndpoint)
-		if err != nil {
-			return availtypes.U128{}, err
-		}
-		a.client = client
-	}
-	var res availtypes.U128
-	meta, err := a.client.RPC.State.GetMetadataLatest()
+func (a *Avail) getBalance() (string, error) {
+	// Initialize the SDK
+	sdk, err := sdk.NewSDK(a.RpcEndpoint)
 	if err != nil {
-		return res, err
+		log.Fatalf("Failed to initialize SDK: %v", err)
 	}
 
-	keyringPair, err := signature.KeyringPairFromSecret(a.Mnemonic, keyringNetworkID)
+	// Get the latest block hash
+	latestBlockHash, err := sdk.Client.BestBlockHash()
 	if err != nil {
-		return res, err
+		log.Fatalf("Failed to get latest block hash: %v", err)
 	}
-	key, err := availtypes.CreateStorageKey(meta, "System", "Account", keyringPair.PublicKey)
+
+	// Initialize the block storage
+	blockStorage, err := sdk.Client.StorageAt(prim.Some(latestBlockHash))
 	if err != nil {
-		return res, err
+		log.Fatalf("Failed to get block storage: %v", err)
 	}
 
-	var accountInfo availtypes.AccountInfo
-	ok, err := a.client.RPC.State.GetStorageLatest(key, &accountInfo)
+	// Create the account ID
+	accountId, err := primitives.NewAccountIdFromAddress(a.AccAddress)
 	if err != nil {
-		return res, err
+		log.Fatalf("Failed to convert address: %v", err)
 	}
-	if !ok {
-		return res, fmt.Errorf("account %s not found", keyringPair.Address)
+
+	// Fetch the account data
+	storage := syPallet.StorageAccount{}
+	val, err := storage.Fetch(&blockStorage, accountId)
+	if err != nil {
+		log.Fatalf("Failed to fetch account: %v", err)
 	}
 
-	return accountInfo.Data.Free, nil
+	// Log the account data
+	fmt.Println("Free Balance: ", val.Value.AccountData.Free.ToHuman())
+	fmt.Println("Reserved Balance: ", val.Value.AccountData.Reserved.ToHuman())
+	fmt.Println("Frozen Balance: ", val.Value.AccountData.Frozen.ToHuman())
```

Comment:
I don't think we/the user care about these two values

Use balance output like we have in other places, eg:

https://github.com/dymensionxyz/roller/blob/main/utils/sequencer/balance.go#L67-L73